feature/onboarding/welcome/src/main/kotlin/app/k9mail/feature/onboarding/welcome/ui/WelcomeContent.kt

```diff
@@ -43,47 +46,51 @@ internal fun WelcomeContent(
     showImportButton: Boolean,
     modifier: Modifier = Modifier,
 ) {
-    Surface(
-        modifier = modifier,
-    ) {
-        ResponsiveContent {
-            LazyColumnWithHeaderFooter(
-                modifier = Modifier.fillMaxSize(),
-                footer = {
-                    WelcomeFooter(
-                        showImportButton = showImportButton,
-                        onStartClick = onStartClick,
-                        onImportClick = onImportClick,
-                        modifier = Modifier
-                            .fillMaxWidth()
-                            .padding(top = MainTheme.spacings.quadruple),
-                    )
-                },
-                verticalArrangement = Arrangement.SpaceEvenly,
-            ) {
-                item {
-                    WelcomeLogo(
-                        modifier = Modifier
-                            .defaultItemModifier()
-                            .padding(top = MainTheme.spacings.double),
-                    )
-                }
-                item {
-                    WelcomeTitle(
-                        title = appName,
-                        modifier = Modifier.defaultItemModifier(),
-                    )
-                }
-                item {
-                    WelcomeMessage(
-                        modifier = Modifier.defaultItemModifier(),
-                    )
-                }
+
+    Surface(modifier = modifier) {
+        Box(Modifier.fillMaxSize()) {
+            ResponsiveContent {
+                LazyColumnWithHeaderFooter(
+                    verticalArrangement = Arrangement.SpaceEvenly,
+                    header = {
+                        WelcomeHeaderSection()
+                    },
+                    footer = {
+                        WelcomeFooterSection(
+                            showImportButton = showImportButton,
+                            onStartClick = onStartClick,
+                            onImportClick = onImportClick,
+                        )
+                    },
+                    content = {
+                        item {WelcomeTitleItem(title = appName)}
+                        item {WelcomeMessageItem()}
+                    },
+                )
             }
+
+
+
         }
     }
 }
 
+@Composable
+private fun WelcomeHeaderSection(
+
```

Comment:
We always add the Modifier to composables. The title could be added here.

```suggestion
    title: String,
    modifier: Modifier = Modifier,
```